src/js/components/homepage/visualizations/treemap/TreeMapTooltip.jsx

```diff
@@ -0,0 +1,121 @@
+/**
+ * TreeMapTooltip.jsx
+ * Created by Emily Gullo 04/28/2017
+ */
+
+import React from 'react';
+
+const propTypes = {
+    name: React.PropTypes.string,
+    value: React.PropTypes.string,
+    description: React.PropTypes.string,
+    height: React.PropTypes.number,
+    x: React.PropTypes.number,
+    y: React.PropTypes.number,
+    width: React.PropTypes.number,
+    showSub: React.PropTypes.bool,
+    percentage: React.PropTypes.string
+};
+
+export default class TreeMapTooltip extends React.Component {
+    componentDidMount() {
+        this.positionTooltip();
+    }
+    componentDidUpdate() {
+        this.positionTooltip();
+    }
+
+    positionTooltip() {
+        // we need to wait for the tooltip to render before we can full position it due to its
+        // dynamic width
+        const tooltipWidth = this.div.offsetWidth;
```

Comment:
This will be handled with a separate PR to come including the new mobile functionality requested by designer.